internal/scheduler/testfixtures/testfixtures.go

```diff
@@ -790,7 +790,7 @@ func Test1Node32CoreExecutor(executorId string) *schedulerobjects.Executor {
 func MakeTestQueue() queue.Queue {
 	return queue.Queue{
 		Name:           TestQueue,
-		PriorityFactor: 0.01,
+		PriorityFactor: 100,
```

Comment:
This had also changed in the previous PR.  PriorityFactors are generally > 1 and weights (i.e. 1/PF) are generally fractional.  For all tests the exact value ere doesn't matter, but semantically it makes more sense to change this back to how is was previously.